DC-SLES-virtualization-best-practices

```diff
@@ -5,7 +5,7 @@
 ##
 ## Basics
 MAIN="MAIN.SLEDS.xml"
-ROOTID=article-vt-best-practices
+ROOTID=article-virtualization
```

Comment:
This one is inconsistent with the others, which are named just `art-...` not `article-...`. However, from the standpoint of using complete English words, I like this version better.

Reply:
I was wondering whether to stick with 'art' or move to 'article', too. Probably will do the latter.